### PR TITLE
Fix class not def found issue

### PR DESCRIPTION
**Pull request by Nirothipan**

### CODE REVIEW

File: components/org.wso2.carbon.micro.integrator.initializer/src/main/java/org/wso2/carbon/micro/integrator/initializer/ServiceBusInitializer.java [hunk 1]
```diff
@@ -217,7 +218,7 @@ protected void deactivate(ComponentContext ctxt) {
 
     private void initPersistence(SynapseConfigurationService synCfgSvc, String configName) throws AxisFault {
         // Initialize the mediation persistence manager if required
-        ServerConfiguration serverConf = ServerConfiguration.getInstance();
+        CarbonServerConfigurationService serverConf = CarbonServerConfigurationService.getInstance();
```
Comment (isudana) [inline components/org.wso2.carbon.micro.integrator.initializer/src/main/java/org/wso2/carbon/micro/integrator/initializer/ServiceBusInitializer.java]: Can we not use Carbon word here?

Comment (isudana) [reply components/org.wso2.carbon.micro.integrator.initializer/src/main/java/org/wso2/carbon/micro/integrator/initializer/ServiceBusInitializer.java]: and is it an osgi service?

Comment (Nirothipan) [reply components/org.wso2.carbon.micro.integrator.initializer/src/main/java/org/wso2/carbon/micro/integrator/initializer/ServiceBusInitializer.java]: Used as we are reading configs from carbon.xml

Comment (Nirothipan) [reply components/org.wso2.carbon.micro.integrator.initializer/src/main/java/org/wso2/carbon/micro/integrator/initializer/ServiceBusInitializer.java]: > is it an osgi service?

yes

**Comment by sdkottegoda** [comment]
Closing since the fixes are duplicated in https://github.com/wso2/micro-integrator/pull/678

**Pull request closed without merging**